packages/integration-tests/src/ui-helpers/index.ts

```diff
@@ -146,3 +150,43 @@ export const expectToClickSidebarMenu = async (page: Page, menuText: string) =>
 export const getInputValue = async (input: ElementHandle<HTMLInputElement>) => {
   return input.evaluate((element) => element.value);
 };
+
+/**
+ * Setup the email connector and update the sign-in experience to the following:
+ *
+ * - Sign-in and register mode
+ * - Use username and password to sign-up
+ * - Use username or email to sign-in
+ * - Email sign-in can use verification code
+ *
+ * @param passwordPolicy The password policy to partially update the existing one.
+ */
+export const setupStandardExperience = async (passwordPolicy?: PartialPasswordPolicy) => {
```

Comment:
nit: not sure on this "standard" experience, could we have a better name? Or can we have agreements on the meaning of "standard".

Reply:
you are right. let me think about it.

Reply:
updated to `setupUsernameAndEmailExperience` to be more specific